kernel/build/pkg.yaml

```diff
@@ -43,14 +43,6 @@ finalize:
   - from: /src/.config
     to: config-{{ if eq .ARCH "aarch64"}}arm64{{ else if eq .ARCH "x86_64" }}amd64{{ else }}unsupported{{ end }}
   {{ else }}
-  - from: /src
-    to: /src
-  - from: /toolchain
-    to: /toolchain
-  - from: /usr
-    to: /usr
-  - from: /bin
-    to: /bin
-  - from: /lib
-    to: /lib
+  - from: /
+    to: /
```

Comment:
Do it this way to avoid losing symlinks. Anyway, these are not released not reproducibility-checked, so it's okay to just pass this stage to the next part as-is

Reply:
but this would mean we copy a large set of files un-necessarily

Reply:
nevermind we were doing the same earlier